client-tez/src/main/java/org/apache/tez/runtime/library/common/shuffle/orderedgrouped/RssShuffleScheduler.java

```diff
@@ -0,0 +1,1690 @@
+     */
+    this.maxTaskOutputAtOnce = Math.max(1, Math.min(75, conf.getInt(
+            TezRuntimeConfiguration.TEZ_RUNTIME_SHUFFLE_FETCH_MAX_TASK_OUTPUT_AT_ONCE,
+            TezRuntimeConfiguration.TEZ_RUNTIME_SHUFFLE_FETCH_MAX_TASK_OUTPUT_AT_ONCE_DEFAULT)));
+
+    this.skippedInputCounter = inputContext.getCounters().findCounter(TaskCounter.NUM_SKIPPED_INPUTS);
+    this.firstEventReceived = inputContext.getCounters().findCounter(TaskCounter.FIRST_EVENT_RECEIVED);
+    this.lastEventReceived = inputContext.getCounters().findCounter(TaskCounter.LAST_EVENT_RECEIVED);
+    this.compositeFetch = ShuffleUtils.isTezShuffleHandler(conf);
+
+    pipelinedShuffleInfoEventsMap = Maps.newConcurrentMap();
+
+    this.storageType = conf.get(RssTezConfig.RSS_STORAGE_TYPE, RssTezConfig.RSS_STORAGE_TYPE_DEFAULT_VALUE);
+    String readBufferSize = conf.get(RssTezConfig.RSS_CLIENT_READ_BUFFER_SIZE,
+        RssTezConfig.RSS_CLIENT_READ_BUFFER_SIZE_DEFAULT_VALUE);
+    this.readBufferSize = (int) UnitConverter.byteStringAsBytes(readBufferSize);
+    this.partitionNumPerRange = conf.getInt(RssTezConfig.RSS_PARTITION_NUM_PER_RANGE,
+        RssTezConfig.RSS_PARTITION_NUM_PER_RANGE_DEFAULT_VALUE);
+
+    LOG.info("RSSShuffleScheduler running for sourceVertex: "
+            + inputContext.getSourceVertexName() + " with configuration: "
+            + "maxFetchFailuresBeforeReporting=" + maxFetchFailuresBeforeReporting
+            + ", reportReadErrorImmediately=" + reportReadErrorImmediately
+            + ", maxFailedUniqueFetches=" + maxFailedUniqueFetches
+            + ", abortFailureLimit=" + abortFailureLimit
+            + ", maxTaskOutputAtOnce=" + maxTaskOutputAtOnce
+            + ", numFetchers=" + numFetchers
+            + ", hostFailureFraction=" + hostFailureFraction
+            + ", minFailurePerHost=" + minFailurePerHost
+            + ", maxAllowedFailedFetchFraction=" + maxAllowedFailedFetchFraction
+            + ", maxStallTimeFraction=" + maxStallTimeFraction
+            + ", minReqProgressFraction=" + minReqProgressFraction
+            + ", checkFailedFetchSinceLastCompletion=" + checkFailedFetchSinceLastCompletion
+            + ", storyType=" + storageType + ", readBufferSize=" + this.readBufferSize
+            + ", partitionNumPerRange=" + partitionNumPerRange);
+  }
+
+  @Override
+  public void start() throws Exception {
+    int shuffleId = InputContextUtils.computeShuffleId(this.inputContext);
+    TezTaskAttemptID tezTaskAttemptID = InputContextUtils.getTezTaskAttemptID(this.inputContext);
+    this.partitionToServers = UmbilicalUtils.requestShuffleServer(
+        inputContext.getApplicationId(), conf, tezTaskAttemptID, shuffleId);
+
+    shuffleSchedulerThread = Thread.currentThread();
+    RssShuffleSchedulerCallable rssShuffleSchedulerCallable = new RssShuffleSchedulerCallable();
+    rssShuffleSchedulerCallable.call();
+  }
+
+  @Override
+  @SuppressWarnings("NN_NAKED_NOTIFY")
```

Comment:
```suggestion
  @SuppressFBWarnings("NN_NAKED_NOTIFY")
```

I think we should use `SuppressFBWarnings` here instead `SuppressWarnings`, and the `SuppressFBWarnings` annotation on class should be removed.

Reply:
这行代码不生成：”EQ_COMPARETO_USE_OBJECT_EQUALS“ 异常，只生成” NN_NAKED_NOTIFY“异常。
所以我认为加上”NN_NAKED_NOTIFY“就行了。

Reply:
> I think we should use `SuppressFBWarnings` here instead `SuppressWarnings`, and the `SuppressFBWarnings` annotation on class should be removed.

There may be error when add to other place.

![image](https://github.com/apache/incubator-uniffle/assets/9440617/c6c6da95-d0e4-4deb-a2a5-c4bedb296f5c)

![image](https://github.com/apache/incubator-uniffle/assets/9440617/bd60ff09-f44b-426e-a426-8af9a4a71abd)

Reply:
It's Suppress**FB**Warnings, not SuppressWarnings